apps/examples/app/components-v2/Divider.tsx

```diff
@@ -0,0 +1,49 @@
+import { Divider, Box, Text, SafeAreaBox } from '@ficus-ui/native';
+import ExampleSection from '@/src/ExampleSection';
+
+const DividerComponent = () => {
+  return (
+    <SafeAreaBox>
+      <Text mx="xl" fontSize="4xl">
+        Divider component
+      </Text>
+      <Box>
+        <ExampleSection name="horizontal">
+          <Text>Test text</Text>
+          <Divider my="md" orientation="horizontal"  />
+          <Text>Test text</Text>
+        </ExampleSection>
+
+        <ExampleSection name="vertical">
+          <Box h={40} flexDirection="row">
+            <Divider orientation="vertical" />
+            <Box m="md">
+              <Text>Test text</Text>
+            </Box>
+          </Box>
+        </ExampleSection>
+
+        <ExampleSection name="custom color">
+          <Text>Test text</Text>
+          <Divider bg="blue" my="md" />
```

Comment:
I'm a bit surprised that is `bg` props that define the color of the Divider, that doesn't looks very natural to me. I think that `color` could sounds more natural?